packages/core/saved-objects/core-saved-objects-api-server/src/saved_objects_client.ts

```diff
@@ -412,4 +412,9 @@ export interface SavedObjectsClientContract {
     spacesToRemove: string[],
     options?: SavedObjectsUpdateObjectsSpacesOptions
   ): Promise<SavedObjectsUpdateObjectsSpacesResponse>;
+
+  /**
+   * Returns namespace associated with the client, if any.
+   */
```

Comment:
NIT: (and because I'm not even sure myself that early in the morning) we may want to explain/document what the expected return is when we are in the `default` namespace (`default` or `undefined`?)